### ISSUE TITLE
where is disparity?

### ISSUE THREAD

**Issue by lunana123** [DavyNeven/fastSceneUnderstanding#14]
![image](https://user-images.githubusercontent.com/41152300/57995476-decb8700-7af4-11e9-88b1-a80287fd2722.png)
![image](https://user-images.githubusercontent.com/41152300/57995487-ed19a300-7af4-11e9-9cd0-633b6aa9ae2e.png)
it can't download

**Comment by gxtron** [comment]
Write an email to [mail@cityscapes-dataset.net](url) with your request for the disparity datasets. I used the same email address as my Cityscapes account

**Comment by JadTawil-theonly** [comment]
exactly, which links am i downloading from cityscapes? https://www.cityscapes-dataset.com/downloads/
